JuiceMaker/JuiceMaker/Enum/JuiceRecipe.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  JuiceRecipe.swift
+//  JuiceMaker
+//
+//  Created by 권태호 on 05/12/2023.
+//
+
+import Foundation
+
+// JuiceRecipe 열거형 정의
+enum JuiceRecipe {
+    case strawberry
+    case banana
+    case kiwi
+    case pineapple
+    case strawberryBanana
+    case mango
+    case mangoKiwi
+    
+    var name: String {
+        switch self {
+        case .strawberry: return "딸기쥬스"
+        case .banana: return "바나나쥬스"
+        case .kiwi: return "키위쥬스"
+        case .pineapple: return "파인애플쥬스"
+        case .strawberryBanana: return "딸바쥬스"
+        case .mango: return "망고쥬스"
+        case .mangoKiwi: return "망고키위쥬스"
+        }
+    }
```

Comment:
```suggestion
     var name: String {
        switch self {
        case .strawberry:
            return "딸기쥬스"
        case .banana:
            return "바나나쥬스"
        case .kiwi:
            return "키위쥬스"
        case .pineapple:
            return "파인애플쥬스"
        case .strawberryBanana:
            return "딸바쥬스"
        case .mango:
            return "망고쥬스"
        case .mangoKiwi:
            return "망고키위쥬스"
        }
    }
```
이건 정말 개인적인 부분이지만 저는 이렇게 줄바꿈 되어있는게 가독성이 더 좋다고 생각하는데 어떠신가요?

Reply:
네 줄바꿈을 해주는 것이 훨씬 가독성이 좋은 것 같습니다 ! 수정 완료 했습니다.
6112bb1b1a52d44e0f24a3fef52e62c030416d9d